contracts/marketplace/CruzoMarket.sol

```diff
@@ -10,6 +10,16 @@ import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 
 import "../transfer-proxy/ITransferProxy.sol";
 
+error ErrInvalidAmount();
+error ErrAlredyOpen();
```

Comment:
Sorry, just a small thing to fix, should be: `ErrAlreadyOpen`

Reply:
fixed in 41d00f4